recipes/loos/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "loos" %}
+{% set version = "4.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GrossfieldLab/loos/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a0c4a0c9ed367a1b8cb6ed22fe08e24a8b4a78785d072e8be96013c9839f6671
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - swig
+  host:
+    - python
+    - pip
+    - numpy
+    - boost-cpp
+    - openblas
+    - libnetcdf <4.9
+    - lapack
+    - eigen
+    
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - scikit-learn
+    - boost-cpp
+    - libopenblas
+    - {{ pin_compatible('libnetcdf') }} 
+    - lapack
+    - eigen
```

Comment:
AFAIK `eigen` is a C++ template-only library and thus should not be needed at runtime.
```suggestion
```

Reply:
Thanks for getting back to us. These suggestions all look pretty reasonable -- we'll test and resubmit soon.

Cheers,

Alan